rust/doc/openapi.yml

```diff
@@ -101,6 +101,26 @@ paths:
       responses:
         "200":
           description: "Ok"
+  /health/performance:
```

Comment:
Didn't we want to hide that endpoint because it is GOS specific?

Reply:
ups!